Assets/_BForBoss/Scripts/Managers/WorldManager.cs.meta

```diff
@@ -4,7 +4,7 @@ MonoImporter:
   externalObjects: {}
   serializedVersion: 2
   defaultReferences: []
-  executionOrder: 0
```

Comment:
I set the execution order of WorldManager to be the lowest, since we want everything else to load in first, before the game loads. 

And it was causing a bug where Graphy (The thing that shows FPS) was loading After WorldManager in builds